src/mongocrypt-kek.c

```diff
@@ -39,6 +39,10 @@
  *    endpoint: <optional string>
  * Local
  *    provider: "local"
+ * KMIP
+ *    provider: "kmip"
+ *    endpoint: <optional string>
+ *    keyId: <optional string>
```

Comment:
```suggestion
 *    keyId: <optional string>
 *    endpoint: <optional string>
```
One more minor reordering.

Reply:
Thank you!